agents-for-bedrock/use-case-examples/text-2-sql-agent-cdk-enhanced/text_to_sql_openapi_schema.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
we should consider updating it to function definitions as it makes the agent creation much simpler

Reply:
yes, lets do for next revision, for now it works perfectly.